tutorials/plot_morph_stc.py

```diff
@@ -100,12 +100,6 @@
 
     - Save a SourceMorph object to disk
 
-Here is an example result for morphing **volume** source estimates:
-
-:ref:`sphx_glr_auto_examples_inverse_plot_morph_volume_stc.py`
-
-.. image:: ../../_images/sphx_glr_plot_morph_volume_stc_001.png
```

Comment:
why did you remove this? and same remark just below?

Reply:
It does not add much understanding, people should just go to the proper tutorial.

And from a practical standpoint, it emits a sphinx warning any time doing a partial build, so it makes it so we can't get the benefit of turning warnings into errors in general.